playwright/plugins/homeserver/synapse/templates/membership-on-events/README.md

```diff
@@ -0,0 +1 @@
+A synapse configured with MSC4115 (user membership in events) support
```

Comment:
Somewhat wondering if we couldn't just set `msc4115_membership_on_events: true` on the default template, rather than have all this copypasta.